src/pages/Transaction/TransactionComp/TransactionComp.tsx

```diff
@@ -18,18 +24,89 @@ const handleCellbaseInputs = (inputs: Cell[], outputs: Cell[]) => {
 }
 
 export const TransactionComp = ({ transaction }: { transaction: Transaction }) => {
-  const { transactionHash, displayInputs, displayOutputs, blockNumber, isCellbase } = transaction
+  const txHash = transaction.transactionHash
+  const location = useLocation()
+  const history = useHistory()
+  const queryParams = new URLSearchParams(location.search)
+  const inputCellsPageNumber = queryParams.get('inputCellsPageNumber')
+  const inputCellsPageSize = queryParams.get('inputCellsPageSize')
+  const outputCellsPageNumber = queryParams.get('outputCellsPageNumber')
+  const outputCellsPageSize = queryParams.get('outputCellsPageSize')
```

Comment:
These can be simplified by `useSearchParams`, e.g. https://github.com/nervosnetwork/ckb-explorer-frontend/blob/develop/src/pages/NftCollections/index.tsx#L22